servicecontroller.go

```diff
@@ -60,37 +58,44 @@ func (sc *ServiceController) Run(stopC chan struct{}) error {
 				close(sc.stopC)
 				sc.stopC = make(chan struct{})
 				sc.endpointScraper = NewEndpointScraper(sc.kubeCli, sc.namespace, srv, sc.logger)
-				sc.endpointScraper.Run(sc.stopC)
+				go sc.endpointScraper.Run(sc.stopC)
+				go func() {
+					select {
```

Comment:
Pourquoi le `select`, si je comprends bien `stopC` sera jamais clos avant.

Reply:
the problem is `NewServiceKiller().Start()` don't have a stop channel
I try to find a way to use `stopC` to kill this start. it may be a bad way to do this.